packages/travix-ui-kit/components/badge/badge.md

```diff
@@ -99,13 +100,34 @@ Position top:
             background
           </Checkbox>
         </div>
+        
+        <div style={{"margin-left":"10px"}}>
+          <h3>States:</h3>
```

Comment:
What do you think to use "Size" as the title here?

Reply:
agree, will add